UnicornRender/video/source/vulkan/ShaderProgram.cpp

```diff
@@ -26,7 +26,7 @@ namespace unicorn
 
                 if (!simpleVertShaderHandler.IsValid() || !simpleFragShaderHandler.IsValid())
                 {
-                    LOG_ERROR("Can't find shaders!");
+                    spdlog::get(utility::Logger::LoggerName())->error("Can't find shaders!");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.